aspnetcore/fundamentals/use-http-context.md

```diff
@@ -185,7 +185,7 @@ This article primarily discusses using `HttpContext` in request and response flo
 
 The following sample logs GitHub branches when requested from the `/branch` endpoint:
 
-[!code-csharp[](~/fundamentals/http-context/samples/6.x/HttpContextInBackgroundThread/Program.cs?highlight=26-45)]
```

Comment:
I noticed the highlight was a line off, so I also changed it. I can revert this if needed.

Old:

![image](https://github.com/dotnet/AspNetCore.Docs/assets/28659384/366d5477-835d-46b5-9c90-6e8f02cc5774)


New:

![image](https://github.com/dotnet/AspNetCore.Docs/assets/28659384/ba67f94f-dec5-4f54-bc0b-ceca77f0224d)